app/routes/board_route.py

```diff
@@ -1,67 +1,78 @@
 from flask import Blueprint, make_response, abort, request
-
-from app import db, Card
+from app import db
 from app.models.board import Board
+from app.models.card import Card
 from app.routes.routes_utilities import validate_model, create_model
 
-board_bp = Blueprint("board_bp", __name__,url_prefix="/boards")
+bp = Blueprint("board_bp", __name__,url_prefix="/boards")
 
 
-@board_bp.post("")
+@bp.post("")
 def create_board():
     response_data = request.get_json()
-    if not response_data.get("title"):
-        abort(make_response({"details": "Invalid data"}, 400))
+    # if not response_data.get("title"):
```

Comment:
Is there a reason to remove it?

Reply:
The create_model() function already has error handling inside. So this would just be doing it twice.